dea_waterbodies/make_time_series.py

```diff
@@ -165,28 +165,63 @@ def get_shapes(config_dict: dict,
 
 @click.command()
 @click.argument('ids', required=False, default='')
-@click.option('--config', '-c', type=click.Path(), default=None)
-@click.option('--shapefile', type=click.Path(), default=None)
-@click.option('--start', type=str, default=None)
-@click.option('--end', type=str, default=None)
-@click.option('--size', type=click.Choice(['ALL', 'SMALL', 'HUGE']),
-              default='ALL')
-@click.option('--missing-only/--not-missing-only', default=False)
-@click.option('--skip', type=click.Path(), default=None)
+@click.option('--config', '-c', type=click.Path(), default=None,
+              help='Path to a config.ini file. A config file can be provided '
+              'instead of using the command line options provided.')
+@click.option('--shapefile', type=click.Path(), default=None,
+              help='REQUIRED. Path to the waterbody polygon shapefile (or '
```

Comment:
--shapefile isn't required if --config is specified.

Reply:
Added.